### PR TITLE
Add Recruitee.com

### PR DESCRIPTION
**Pull request by bogumilmazur**

### CODE REVIEW

File: img/r/recruitee.com.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (Carlgo11) [inline img/r/recruitee.com.svg]: Please use a minimized SVG.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M25.77 13.7h-.06a10.36 10.36 0 00-19.63 0 6.04 6.04 0 00-4.09 10.35c.13.12.32.13.46.03a8.53 8.53 0 012.57-1.42c.09-.03.55-.28.3-.96a4.28 4.28 0 01-1.17-2.86c0-1.65 1.1-2.52 2.37-2.52 1.28 0 2.38.87 2.38 2.52 0 1.08-.5 2.17-1.16 2.85-.25.77.15.95.22.97.22.07.45.16.67.25.12.05.14.2.06.28a4.9 4.9 0 00-1.15 1.85c-.06.17.15.31.28.19.45-.42 1.04-.92 1.8-1.43.98-.73 2.33-1.38 3.57-1.83.17-.05.96-.49.52-1.65a7.36 7.36 0 01-2-4.91c0-2.85 1.88-4.34 4.08-4.34s4.09 1.5 4.09 4.34c0 1.85-.87 3.73-2 4.9-.43 1.31.27 1.63.37 1.66 1.27.42 2.6 1.03 3.6 1.74.84.51 1.51 1.04 2.03 1.5.13.11.33-.02.28-.2a4.87 4.87 0 00-1.17-1.86.17.17 0 01.05-.29l.57-.22c.1-.04.56-.29.3-.97a4.3 4.3 0 01-1.17-2.86c0-1.66 1.1-2.53 2.38-2.53 1.28 0 2.39.87 2.39 2.53a4.3 4.3 0 01-1.17 2.85c-.25.77.16.96.22.98 1.06.35 2.21.93 2.72 1.58.13.15.35.17.5.04a6.03 6.03 0 00-4-10.56" fill="#1999E3"/></svg>
</svg>
```

File: entries/r/recruitee.com.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "Recruitee": {
+    "domain": "recruitee.com",
+    "url": "https://recruitee.com/",
```
Comment (Carlgo11) [inline entries/r/recruitee.com.json]: As the error message states, Defining the URL property isn't necessary when it's the same as the default value.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/r/recruitee.com.svg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M25.77 13.7h-.06a10.36 10.36 0 00-19.63 0 6.04 6.04 0 00-4.09 10.35c.13.12.32.13.46.03a8.53 8.53 0 012.57-1.42c.09-.03.55-.28.3-.96a4.28 4.28 0 01-1.17-2.86c0-1.65 1.1-2.52 2.37-2.52 1.28 0 2.38.87 2.38 2.52 0 1.08-.5 2.17-1.16 2.85-.25.77.15.95.22.97.22.07.45.16.67.25.12.05.14.2.06.28a4.9 4.9 0 00-1.15 1.85c-.06.17.15.31.28.19.45-.42 1.04-.92 1.8-1.43.98-.73 2.33-1.38 3.57-1.83.17-.05.96-.49.52-1.65a7.36 7.36 0 01-2-4.91c0-2.85 1.88-4.34 4.08-4.34s4.09 1.5 4.09 4.34c0 1.85-.87 3.73-2 4.9-.43 1.31.27 1.63.37 1.66 1.27.42 2.6 1.03 3.6 1.74.84.51 1.51 1.04 2.03 1.5.13.11.33-.02.28-.2a4.87 4.87 0 00-1.17-1.86.17.17 0 01.05-.29l.57-.22c.1-.04.56-.29.3-.97a4.3 4.3 0 01-1.17-2.86c0-1.66 1.1-2.53 2.38-2.53 1.28 0 2.39.87 2.39 2.53a4.3 4.3 0 01-1.17 2.85c-.25.77.16.96.22.98 1.06.35 2.21.93 2.72 1.58.13.15.35.17.5.04a6.03 6.03 0 00-4-10.56" fill="#1999E3"/></svg>
+</svg>
```
Comment (ApCoder123) [inline img/r/recruitee.com.svg]:
```suggestion
```

There's an extra `</svg>` tag at the end of this file which can be removed.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

File: entries/r/recruitee.com.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "Recruitee": {
+    "domain": "recruitee.com",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://support.recruitee.com/en/articles/5209837-protect-your-account-with-two-factor-authentication-2fa",
```
Comment (ApCoder123) [inline entries/r/recruitee.com.json]:
```suggestion
    "documentation": "https://support.recruitee.com/en/articles/5209837",
```
This URL can be shortened

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**